configfs.c

```diff
@@ -72,6 +72,54 @@ int tcmu_set_control(struct tcmu_device *dev, const char *key, unsigned long val
 	return tcmu_set_cfgfs_str(path, buf, strlen(buf) + 1);
 }
 
+static int tcmu_set_block_netlink(unsigned long val)
+{
+	char path[PATH_MAX];
+
+	snprintf(path, sizeof(path), CFGFS_MOD_PARAM"/block_netlink");
+
+	return tcmu_set_cfgfs_ul(path, val);
+}
+
+static void tcmu_block_netlink(void)
+{
+	tcmu_set_block_netlink(1);
+}
```

Comment:
No need for the wrapper and we will probably never need it, so just move the insides of tcmu_set_block_netlink into this function.

Same with unblock below.